### DISCUSSION TITLE
service exposed outside of the vnet meaning?

### DISCUSSION THREAD

**Discussion by bwiseman** [microsoft/AzureTRE#3620] [Q&A]
Guacamole Virtual Desktop Service - Is the Guacamole service exposed outside of the vnet?

What does this option actually open up/close off?

**Comment by marrobi** [comment]
If you wanted to have a guacamole service that's only accessible from inside  the VNet. Its the web app endpoint. This is rarely used, but had a use case where users would use a VNet peer to access the workspaces.
